server/src/main/java/za/dots/db/DatasourceConnection.java

```diff
@@ -0,0 +1,21 @@
+package za.dots.db;
+
+import java.sql.*;
+
+public class DatasourceConnection {
+
+    Connection connection;
+
+    private String cleanInputValue(String string) {
```

Comment:
This function is not used anywhere

Reply:
To be used. Have not implemented actual inserts yet